internal/service/appstream/stack.go

```diff
@@ -75,7 +75,7 @@ func ResourceStack() *schema.Resource {
 						"settings_group": {
 							Type:         schema.TypeString,
 							Optional:     true,
-							ValidateFunc: validation.StringLenBetween(0, 100),
+							ValidateFunc: validation.StringLenBetween(0, 1000),
```

Comment:
This limit on `settings_group` is correctly set to 100.
https://docs.aws.amazon.com/appstream2/latest/APIReference/API_ApplicationSettings.html#AppStream2-Type-ApplicationSettings-SettingsGroup

I think maybe you meant to change this ValidateFunc?

https://github.com/hashicorp/terraform-provider-aws/blob/f23a9c4ae983e7ca765397474e342db26059a636/internal/service/appstream/stack.go#L128-L133

Reply:
Well, that's embarrassing 🤦